apps/admin/src/dataProvider.js

```diff
@@ -0,0 +1,5 @@
+import simpleRestProvider from "ra-data-simple-rest";
+
+const dataProvider = simpleRestProvider(process.env.API_URL);
```

Comment:
We'll have to use our own dataprovider (for pagination) ... Maybe could you boostrap it ?